modules/wso2is/templates/wso2is.erb

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
```

Comment:
let's rename this as wso2is-service, WDYT?

Reply:
This is removed and only `wso2is.service.erb` will be used to start the server as a service.